src/utils.ts

```diff
@@ -0,0 +1,46 @@
+import { warn, watch } from 'vue'
+import { VueDriftWidgetConfig } from './types'
+
+const inBrowser: boolean = typeof window !== 'undefined'
+
+export const loadScript = (
+  id: string,
+  config: VueDriftWidgetConfig,
+  snippetVersion = '0.3.1',
```

Comment:
how does this affect the version loaded.. it seems to just be the version of the snippet we are using as it is not affecting the javascript file included as src of the script tag?

suggestion: remove because unneeded?

Reply:
this affects the version rendered, not the version loaded. this is used by the widget itself and needs to be set